src/cargo/ops/cargo_new.rs

```diff
@@ -182,3 +202,20 @@ fn global_config(config: &Config) -> CargoResult<CargoNewConfig> {
         version_control: vcs,
     })
 }
+
+#[cfg(test)]
+mod tests {
+    use super::strip_rust_affixes;
+
+    #[test]
+    fn affixes_stripped() {
+        assert_eq!(strip_rust_affixes("rust-foo"), "foo");
+        assert_eq!(strip_rust_affixes("foo-rs"), "foo");
+        assert_eq!(strip_rust_affixes("rs_foo"), "foo");
+        // Only one affix is stripped
```

Comment:
perhaps it shouldn't strip at all if there are multiple matches in this way? it's not clear to me which one would be correct to strip